lib/src/widgets/movie_grid_builder.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:animate_do/animate_do.dart';
+import 'package:flutter/material.dart';
+import '../ui/movie_detail.dart';
+import '../utils/movie_dimensions.dart';
+
+GridView buildList(AsyncSnapshot snapshot) {
```

Comment:
you can make the name of the method a bit more descriptive, also if you are returning a GridView, you should make a reference to that

Reply:
done